ql/src/test/queries/clientpositive/avro_hybrid_mixed_timestamp.q

```diff
@@ -5,13 +5,13 @@ stored as avro;
 
 INSERT INTO hybrid_table VALUES
 ('2012-02-21 07:08:09.123'),
-('2014-02-11 07:08:09.123'),
+('2014-02-11 07:08:09.123456'),
 ('1947-02-11 07:08:09.123'),
-('8200-02-11 07:08:09.123'),
-('1012-02-21 07:15:11.123'),
-('1014-02-11 07:15:11.123'),
+('8200-02-11 07:08:09.123456'),
+('1012-02-21 07:15:11.12345'),
+('1014-02-11 07:15:11.1234'),
 ('0947-02-11 07:15:11.123'),
-('0200-02-11 07:15:11.123');
+('0200-02-11 07:15:11.1234');
```

Comment:
Always refrain from modifying existing tests. Considering adding new tests.

Reply:
Makes sense, added extra inserts to check precision of timestamp